packages/opentelemetry-metrics/README.md

```diff
@@ -11,11 +11,26 @@ OpenTelemetry metrics allow a user to collect data and export it to a metrics ba
 ## Installation
 
 ```bash
-npm install --save @opentelemetry/metrics
+npm install --save @opentelemetry/metrics @opentelemetry/api
 ```
 
 ## Usage
 
+### Install Export Pipeline
+
+It is essential to install export pipeline first before all subsequent meter initialization to get exporters work properly. Here we'll take `ConsoleMetricExporter` for an example.
+
+```js
+import { ConsoleMetricExporter } from '@opentelemetry/metrics'
+import { metrics } from '@opentelemetry/api'
+
+// Install the export pipeline before all subsequent call to metrics.
+ConsoleMetricExporter.installPipeline();
```

Comment:
why is this static, have different name than in prometheus and the flow will also looks differently ?
I think this all should look the same doesn't matter if this is console exporter, prometheus exporter or any other exporter. I think we should have then one method in metrics which will install the `export pipeline` by accepting an exporter as dependency injection.

Reply:
I'll rename those `installExportPipeline` aligned if the above explanation make sense to you.